mantidimaging/gui/windows/recon/presenter.py

```diff
@@ -396,7 +396,7 @@ def completed(task: TaskWorkerThread) -> None:
                     f"\n\n Suggestion: Use crop coordinates to resize the 180 degree projection to "
                     f"({selected_stack.height}, {selected_stack.width})")
             else:
-                cor, tilt = task.result
+                cor, tilt = task.result if task.result is not None else (None, None)
```

Comment:
I'd prefer an assert here. We'd only reach this if both `task.result` and `task.error` were `None`, which would mean something bad was wrong and we'd want to raise that to an error message.

Would it work to restructure the `if` as
```
if task.error is not None:
  ...
elif task.result is not None:
  ...
else:
  raise AssertionError("task in inconsistent state")
```
It would be a bit more readable maybe for humans and type checkers.